.github/renovate.json

```
@@ -7,7 +7,7 @@
     },
     "packageRules": [
       {
-        "matchPackagePrefixes": ["third_party/"],
+        "matchDepNames": ["third_party/ni-apis", "third_party/protobuf"],
         "extends": ["schedule:monthly"]
```

From @dixonjoel in https://github.com/ni/measurement-plugin-labview/pull/582#issuecomment-2322331921

> This is an improvement over getting an update for each commit. So I suggest we check this in for now and discuss further.
>
> @jasonmreding and I discussed this and I think we're of the same mind that we want to just turn off auto-updating altogether for the ni-apis repo (and maybe the protobuf). The reason for this is we don't have an automated way to generate stubs and / or check for out-of-date stubs like Python does. So if we auto-update, we'll have .proto files that don't match the generated code.

This should prevent Renovate from creating PRs until you approve via the Dependency Dashboard.

```suggestion
        "extends": ["schedule:monthly", ":dependencyDashboardApproval"]
```